src/components/ble/BleNus.cpp

```diff
@@ -0,0 +1,92 @@
+#include "BleNus.h"
+
+using namespace Pinetime::Controllers;
+
+constexpr ble_uuid128_t BleNus::nusServiceUuid;
+constexpr ble_uuid128_t BleNus::rxCharacteristicUuid;
+constexpr ble_uuid128_t BleNus::txCharacteristicUuid;
+uint16_t BleNus::attr_read_handle;
+//static uint16_t conn_handle;
```

Comment:
If this is unused it should probably be removed.

Reply:
This is a PR draft to brainstorm, not a PR review. We can deal with details that later. I would appreciate some help/ideas how to integrate this, we can iron all other things later when I chage draft do review.